sdk/identity/sshKey

```diff
@@ -0,0 +1,50 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
feels odd checking in a private key like this, are we going to get yelled at by the credential scanner? Do other languages check in the key this way?

Reply:
I thought this got skipped!

Reply:
I think if you name the private key `ssh.key` it'll be gitignored correctly https://github.com/azure/azure-sdk-for-js/blob/main/.gitignore#L127

You could also add sshKey and the public one here: https://github.com/maorleger/azure-sdk-for-js/blob/main/sdk/identity/identity/.gitignore